src/fhirtypes/StructureDefinition.ts

```diff
@@ -366,6 +367,10 @@ export class StructureDefinition {
    * @returns {ElementDefinition} the new ElementDefinition
    */
   newElement(name = '$UNKNOWN'): ElementDefinition {
+    //Check if there already exists an element that is defined by an ancestor
```

Comment:
Sorry for the super nitpicky comment, but can you add a space after the `//`? I thought we had a lint rule for that, but seems like maybe not.
```suggestion
    // Check if there already exists an element that is defined by an ancestor
```